hypervisor/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0 OR BSD-3-Clause"
 [features]
 kvm = ["kvm-ioctls", "kvm-bindings", "vfio-ioctls/kvm"]
 mshv = ["mshv-ioctls", "mshv-bindings", "vfio-ioctls/mshv", "iced-x86"]
-sev_snp = ["igvm_parser", "igvm_defs"]
+sev_snp = ["igvm_defs", "igvm_parser"]
```

Comment:
Ditto. This is also why I suggested to introduce dependencies at the time when they are used for the first time.